android/expoview/src/main/java/versioned/host/exp/exponent/modules/api/notifications/NotificationsModule.java

```diff
@@ -296,17 +332,15 @@ public void scheduleLocalNotificationWithChannel(final ReadableMap data, final R
           legacyChannelData.toHashMap());
     }
 
-    int notificationId = new Random().nextInt();
+    int notificationId = Math.abs(new Random().nextInt());
```

Comment:
Fun fact: This can still be negative (very unlikely). In case nextInt() returns Integer.MIN_VALUE. It is |MIN_VALUE| = |MAX_VALUE+1| (check Integer range in java), which will lead to Integer being overflowed.
In short: Math.abs(Integer.MIN_VALUE) == Integer.MIN_VALUE :)

The safest way would be to call new Random().nexInt(Integer.MAX_VALUE).